### PR TITLE
[TTAHUB-858] Show and Filter on Objective Topics on Goals and Objective Page

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

~~We need to make sure that we display and filter goals based on the objective topics.~~

For now we will ONLY show AR topics and filter AR topics.

**Note:** Garrett is updating the migration script to move AR topics to the objectives.

## How to test

~~Create a goal with objective topics. These objective topics should show on the Goals and Objectives page. Filtering on these topics should reflect the goals we display.~~

User should be able to see and filter the Goals and Objective tab using AR topics only.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-858

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
Maybe these questions will be solved by updating the migration script, not sure

- How do we create a goal with objective topics?
- Why are the goal topics empty on the RTR?

**Comment by AdamAdHocTeam** [comment]
> Maybe these questions will be solved by updating the migration script, not sure
> 
> * How do we create a goal with objective topics?

I don't think this is supported at the moment. Probably will need the new redesign.

Do we want to still include the AR topics?

> * Why are the goal topics empty on the RTR?

The goal topics on the RTR are empty because the objectives don't have goals. 

Garrett is writing a script to move over AR topics to objectives. So they will be populated.

File: src/scopes/goals/topics.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Op } from 'sequelize';
 import { filterAssociation } from './utils';
 
+/*
```
Comment (jasalisbury) [inline src/scopes/goals/topics.js:4]: I assume these lines will be uncommented after the new AR, RTR, and topic migration are released. Can we tag them somehow so it is easy to find commented code here (and elsewhere in this PR) that will need to be updated once everything is released?

**Review by jasalisbury — APPROVED** [review]
:rocket:

**Pull request merged**